frontend/components/Navbar.tsx

```diff
@@ -2,19 +2,27 @@ import { Divider } from "@material-ui/core";
 import { useSession } from "next-auth/client";
 import Link from "next/link";
 import { useRouter } from "next/router";
-import PropTypes from "prop-types";
-import React from "react";
+import React, { FC } from "react";
 import Image from "react-bootstrap/Image";
 import Nav from "react-bootstrap/Nav";
 import Navbar from "react-bootstrap/Navbar";
 import NavDropdown from "react-bootstrap/NavDropdown";
 import { AUTH_REDIRECT_PATH, handleLogin, handleLogout, LOGIN_PAGE_PATH } from "../auth";
 
-const globalMenuItems = [
+interface GlobalMenuItem {
+  title: string;
+  path: string;
+  reactive?: boolean;
+  submenuItems?: GlobalMenuItemWithoutSubmenuItems[];
+}
+
+type GlobalMenuItemWithoutSubmenuItems = Omit<GlobalMenuItem, "submenuItems">;
```

Comment:
This is verbose (👎) but clear and removes the need to repeat the Omit in a few places (👍).